src/GitBucket.Service.Tests/ReleaseServicetest.cs

```diff
@@ -378,7 +378,6 @@ private static GitBucketDbContext EnsureDbCreated(ReleaseOptions options)
             var highPriority = new Core.Models.Priority
             {
                 Ordering = 0,
-                PriorityId = 0,

```

Comment:
```
X GitBucket.Service.Tests.ReleaseServiceTest.Should_Output_ReleaseNote [90ms]
  Error Message:
   System.InvalidOperationException : The instance of entity type 'Priority' cannot be tracked because another instance with the same key value for {'PriorityId'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.
  Stack Trace:
     at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.ThrowIdentityConflict(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(TKey key, InternalEntityEntry entry, Boolean updateDuplicate)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(TKey key, InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.IdentityMap`1.Add(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.StartTracking(InternalEntityEntry entry)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry.SetEntityState(EntityState oldState, EntityState newState, Boolean acceptChanges, Boolean modifyProperties)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.InternalEntityEntry.SetEntityState(EntityState entityState, Boolean acceptChanges, Boolean modifyProperties, Nullable`1 forceStateWhenUnknownKey)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityGraphAttacher.PaintAction(EntityEntryGraphNode`1 node)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityEntryGraphIterator.TraverseGraph[TState](EntityEntryGraphNode`1 node, Func`2 handleNode)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityEntryGraphIterator.TraverseGraph[TState](EntityEntryGraphNode`1 node, Func`2 handleNode)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.EntityGraphAttacher.AttachGraph(InternalEntityEntry rootEntry, EntityState targetState, EntityState storeGeneratedWithKeySetTargetState, Boolean forceStateWhenUnknownKey)
   at Microsoft.EntityFrameworkCore.DbContext.SetEntityState(InternalEntityEntry entry, EntityState entityState)
   at Microsoft.EntityFrameworkCore.DbContext.SetEntityStates(IEnumerable`1 entities, EntityState entityState)
   at Microsoft.EntityFrameworkCore.DbContext.AddRange(IEnumerable`1 entities)
   at Microsoft.EntityFrameworkCore.Internal.InternalDbSet`1.AddRange(IEnumerable`1 entities)
   at GitBucket.Service.Tests.ReleaseServiceTest.EnsureDbCreated(ReleaseOptions options) in C:\projects\gitbucket-utility\src\GitBucket.Service.Tests\ReleaseServicetest.cs:line 441
   at GitBucket.Service.Tests.ReleaseServiceTest.Should_Output_ReleaseNote() in C:\projects\gitbucket-utility\src\GitBucket.Service.Tests\ReleaseServicetest.cs:line 337
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__139_0(Object state)
[xUnit.net 00:00:01.87]     GitBucket.Service.Tests.ReleaseServiceTest.PullRequest_Already_Exists [FAIL]
```